docs/source/developer/extension_tutorial.rst

```diff
@@ -162,6 +162,7 @@ dependencies and install the extension into the JupyterLab environment.
 
 .. code:: bash
 
+    pip install jupyter_packaging
```

Comment:
An alternative could be to have it as part of the `conda create` command above?

Reply:
Yep. Do you prefer it there? I thought it fit here as an "initial project dependency", but I could also see it up in the conda create above. I don't have a strong preference either way.

Reply:
I moved it up to the conda environment creation.